src/sys/socket/addr.rs

```diff
@@ -676,21 +711,28 @@ impl fmt::Display for UnixAddr {
     }
 }
 
+impl fmt::Debug for UnixAddr {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        fmt::Display::fmt(self, f)
+    }
+}
+
 /*
  *
  * ===== Sock addr =====
  *
  */
 
 /// Represents a socket address
-#[derive(Copy)]
+#[derive(Copy, Debug)]
 pub enum SockAddr {
     Inet(InetAddr),
     Unix(UnixAddr),
     #[cfg(any(target_os = "android", target_os = "linux"))]
     Netlink(NetlinkAddr),
     #[cfg(any(target_os = "ios", target_os = "macos"))]
     SysControl(SysControlAddr),
+    Ether(EtherAddr)
```

Comment:
Please put a comma at the end of this line and this should also have doccomments. It'd be very much appreciated if you could provide doc comments for the other variants here as well.